CatAnalyzer/python/leptonSF_cff.py

```diff
@@ -92,9 +92,79 @@
 )
 
 ## Electron SF from https://twiki.cern.ch/twiki/bin/view/CMS/EgammaIDRecipesRun2?rev=36#Electron_efficiencies_and_scale
-## root file is retrieved from http://fcouderc.web.cern.ch/fcouderc/EGamma/scaleFactors/ichep2016_80X//resultsEleID/runBCD/passingMedium/egammaEffi.txt_SF2D.root
+## root file is retrieved from 
+##  GSF: http://fcouderc.web.cern.ch/fcouderc/EGamma/scaleFactors/ichep2016_80X/resultsGsfID/egammaEffi.txt_SF2D.root
+##   ID: http://fcouderc.web.cern.ch/fcouderc/EGamma/scaleFactors/ichep2016_80X//resultsEleID/runBCD/passingMedium/egammaEffi.txt_SF2D.root
+## Systematic uncertainty of 1% is added for pt>20, 3% for pt<20
 electronSFCutBasedIDMediumWP = cms.PSet(
     pt_bins = cms.vdouble(10.000000,20.000000,30.000000,40.000000,50.000000,200.000000),
+    eta_bins = cms.vdouble(-2.500000,-2.400000,-2.300000,-2.200000,-2.000000,-1.800000,-1.630000,-1.566000,-1.444000,-1.200000,-1.000000,-0.800000,-0.600000,-0.400000,-0.200000,0.000000,0.200000,0.400000,0.600000,0.800000,1.000000,1.200000,1.444000,1.566000,1.630000,1.800000,2.000000,2.200000,2.300000,2.400000,2.500000),
```

Comment:
For an inline note: the eta binning is changed while merging reco SF and id SF, where two SF have different eta binning.
If you feel this is too complicated, you can try little bit different way - apply two SFs separately.